.github/workflows/release_ruleset.yaml

```diff
@@ -101,25 +140,54 @@ jobs:
           RELEASE_ARCHIVE_REGEX="(.zip|.jar|.war|.aar|.tar|.tar.gz|.tgz|.tar.xz|.txz|.tar.xzt|.tzst|.tar.bz2|.ar|.deb)$"
 
           ATTESTATIONS_DIR=$(mktemp --directory)
-          RELEASE_FILES="${{ join(inputs.release_files, ' ') }}"
-          for filename in $RELEASE_FILES; do
+          for filename in $(find release_files/ -type f -printf "%f\n"); do
             if [[ "${filename}" =~ $RELEASE_ARCHIVE_REGEX ]]; then
                 ATTESTATION_FILE="$(basename "${filename}").intoto.jsonl"
                 echo "Writing attestation to ${ATTESTATION_FILE}"
                 cat ${{ steps.attest_release.outputs.bundle-path }} | jq --compact-output > "${ATTESTATIONS_DIR}/${ATTESTATION_FILE}"
             fi
           done
           echo "release_archive_attestations_dir=${ATTESTATIONS_DIR}" >> $GITHUB_OUTPUT
+      - uses: actions/upload-artifact@v4
+        id: upload-attestations
+        with:
+          name: attestations
+          path: ${{ steps.write_release_archive_attestation.outputs.release_archive_attestations_dir }}/*
+
+  release:
```

Comment:
I think we had mentioned optionally doing releases, so you could gate this job with a `if:`

Reply:
I think this is not optional for us

Reply:
Yeah, we always want to run a release when using this workflow.